[2025-05-26T15:11:37.643+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.task1 scheduled__2025-05-26T12:36:00+00:00 [queued]>
[2025-05-26T15:11:37.656+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.task1 scheduled__2025-05-26T12:36:00+00:00 [queued]>
[2025-05-26T15:11:37.657+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 6
[2025-05-26T15:11:37.686+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): task1> on 2025-05-26 12:36:00+00:00
[2025-05-26T15:11:37.695+0000] {standard_task_runner.py:57} INFO - Started process 5128 to run task
[2025-05-26T15:11:37.696+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'task1', 'scheduled__2025-05-26T12:36:00+00:00', '--job-id', '747', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpn6hcygyp']
[2025-05-26T15:11:37.705+0000] {standard_task_runner.py:85} INFO - Job 747: Subtask task1
[2025-05-26T15:11:37.724+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-26T15:11:37.764+0000] {task_command.py:415} INFO - Running <TaskInstance: first_dag.task1 scheduled__2025-05-26T12:36:00+00:00 [running]> on host 4ef62d544527
[2025-05-26T15:11:37.849+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sirtaf' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2025-05-26T12:36:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-26T12:36:00+00:00'
[2025-05-26T15:11:37.854+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-26T15:11:37.856+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello world, this is the first task!']
[2025-05-26T15:11:37.875+0000] {subprocess.py:86} INFO - Output:
[2025-05-26T15:11:37.877+0000] {subprocess.py:93} INFO - Hello world, this is the first task!
[2025-05-26T15:11:37.880+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-26T15:11:37.910+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=task1, execution_date=20250526T123600, start_date=20250526T151137, end_date=20250526T151137
[2025-05-26T15:11:37.969+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-26T15:11:37.996+0000] {taskinstance.py:2776} INFO - 2 downstream tasks scheduled from follow-on schedule check
